executors/kubeexec/kubeexec.go

```diff
@@ -206,6 +223,14 @@ func (k *KubeExecutor) ConnectAndExec(host, namespace, resource string,
 			return nil, fmt.Errorf("User %v credentials not accepted", k.config.User)
 		}
 		clientConfig.BearerToken = token
+	} else if k.config.UseSecrets {
+		tokenBytes, err := ioutil.ReadFile(k.config.TokenFile)
+		if err != nil {
+			logger.Err(err)
+			return nil, fmt.Errorf("Secret token not found in %v", k.config.TokenFile)
```

Comment:
fyi if you saw the email, I have updated the comment to suggest using logger.LogError() instead of logger.Err()